src/pages/utilities/FileSearch.tsx

```diff
@@ -0,0 +1,446 @@
+        </div>
+        {seriesInfo?.Titles.find(x => x.Type === 'Main')?.Name ?? 'Unknown'}
+      </div>
+      <div className="mb-4 flex flex-col gap-y-2">
+        <div className="flex justify-between capitalize">
+          <span className="font-semibold">Episode Name</span>
+        </div>
+        {episodeInfo?.Title ?? 'Unknown'}
+      </div>
+      <div className="mb-4 flex flex-col gap-y-2">
+        <div className="flex justify-between capitalize">
+          <span className="font-semibold">Location</span>
+        </div>
+        {get(file, 'Locations.0.RelativePath', 'Unknown')}
+      </div>
+      <div className="mb-4 flex flex-col gap-y-2">
+        <div className="flex justify-between capitalize">
+          <span className="font-semibold">Video</span>
+        </div>
+        {videoInfo.join(' | ')}
+      </div>
+      <div className="mb-4 flex flex-col gap-y-2">
+        <div className="flex justify-between capitalize">
+          <span className="font-semibold">Audio</span>
+        </div>
+        {audioInfo.join(' | ')}
+      </div>
+      <div className="mb-4 flex flex-col gap-y-2">
+        <div className="flex justify-between capitalize">
+          <span className="font-semibold">Hash</span>
+        </div>
+        {get(file, 'Hashes.ED2K', '')}
+      </div>
+      <div className="mb-4 flex flex-col gap-y-2">
+        <div className="flex justify-between capitalize">
+          <span className="font-semibold">CRC</span>
+        </div>
+        {get(file, 'Hashes.CRC32', '')}
+      </div>
+      <div className="mb-4 flex flex-col gap-y-2">
+        <div className="flex justify-between capitalize">
+          <span className="font-semibold">SHA1</span>
+        </div>
+        {get(file, 'Hashes.SHA1', '')}
+      </div>
+    </TransitionDiv>
+  );
+};
+
+const FileSearch = () => {
+  const [search, setSearch] = useState('');
+  const debounceSearch = useDebounce(search, 250);
+  const { data: fileResults, fetchNextPage, isFetchingNextPage } = useFilesInfiniteQuery({
+    include: ['XRefs'],
+    sortOrder: [-FileSortCriteriaEnum.ImportedAt],
+    pageSize: 50,
+  }, debounceSearch);
+  const [files, fileCount] = useFlattenListResult<FileType>(fileResults);
+
+  const {
+    handleRowSelect,
+    rowSelection,
+    selectedRows,
+    setRowSelection,
+  } = useRowSelection<FileType>(files);
+
+  const [viewIndex, setViewIndex] = useState(0);
+
+  const onNextView = useCallback(() => {
+    setViewIndex((prev) => {
+      if (prev + 1 >= selectedRows.length) return 0;
+
+      return prev + 1;
+    });
+  }, [selectedRows]);
+
+  const onPrevView = useCallback(() => {
+    setViewIndex((prev) => {
+      if (prev - 1 < 0) return selectedRows.length - 1;
+
+      return prev - 1;
+    });
+  }, [selectedRows]);
+
+  const selectedId = useMemo(() => selectedRows[viewIndex]?.ID, [selectedRows, viewIndex]);
+
+  const columns = useMemo<UtilityHeaderType<FileType>[]>(
+    () => staticColumns,
+    [],
+  );
```

Comment:
You can just pass `staticColumns` to the `UtilitiesTable` if you aren't going to add/remove anything. This is not needed